nbs/15_gsa_gls/30-real_phenotypes/05-group_gwas_by_n_variants.ipynb

```diff
@@ -0,0 +1,1599 @@
+{
```

Comment:
<p>what is rapid and astle?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/phenoplier/pull/65/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Reply:
They are cohorts (sets of GWAS results with the same set of variants). I updated the description.